keel-ec2-plugin/src/test/kotlin/com/netflix/spinnaker/keel/ec2/resource/ClusterHandlerTests.kt

```diff
@@ -198,6 +221,40 @@ internal class ClusterHandlerTests : JUnit5Minutests {
           health.terminationPolicies.map(TerminationPolicy::toString).toSet(),
           subnets.map(Subnet::id).joinToString(",")
         ),
+        listOf(ScalingPolicy(
+          "$name-v$sequence",
+          "$name-target-tracking-policy",
+          "TargetTrackingScaling",
+          300,
+          null,
+          null,
+          null,
+          null,
+          null,
```

Comment:
This is pretty hard to read without the argument names. Could you please add them?

Reply:
Will do!